tox.ini

```diff
@@ -0,0 +1,14 @@
+[tox]
+skipsdist = true
+envlist = py{27,35}
+
+[testenv]
+passenv = DISPLAY
+deps =
+	pytest==3.0.3
+	pytest-selenium==1.4.0
+	selenium==3.0.0b2
+commands = pytest --driver Firefox foxpuppet/tests
```

Comment:
Hm, now {posargs} is gone. Was that expected? Also the test path appears twice now. I think that this not necessary, or?